### ISSUE TITLE
[feature]: Navigate to module from ticket

### ISSUE THREAD

**Issue by omarin-disco** [makeplane/plane#5527]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

When in any ticket view (list of tickets or a single ticket), I should be able to click any module associated with a ticket and be taken to that module rather than this toggling that module being assigned to the ticket.

At present, there's no way to navigate from a ticket to its associated module(s). In the ticket details view, each module listed has two ways to remove it. You can click the X next to the module, or you can click the module and then click the name in the dorpdown. Instead of bringing up a dropdown when clicking the module name, you should be taken to that module. Clicking on the modules field should bring up the dropdown to select modules. Additionally, any modules you select could be added to the field and removed from the dropdown, giving you a consistent way to remove modules (always clicking the X), rather than having a checkmark next to modules that are listed and selected.

### Why should this be worked on?

Having modules listed in the field and in the dropdown is duplicative. Removing modules with an X or by clicking module name in the dropdown is duplicative. Clicking the module name to navigate is more intuitive, and is an interaction that is currently missing from Plane.

**Comment by sabgaby** [comment]
Yes Yes Yes!!! I just checked Linear's UX, and apparently ⌘+Click on the issues Projects (/Modules), it opens it up in a new tab
